feature/experimental/isis/otg_tests/isis_hello_interval_and_multiplier_configuration_test/README.md

```diff
@@ -0,0 +1,55 @@
+# RT-2.16: IS-IS hello-interval and multiplier Configuration Test
+
+## Summary
+
+IS-IS hello-interval and multiplier Configuration Test
+
+## Procedure
+
+* Topology: ATE-port1<—> DUT–port1
+    * Configure IS-IS for ATE port-1 and DUT port-1. 
+    * Establish basic adjacency
+* Baseline Configuration:
+    * Set the hello-interval to a standard value (e.g., 10 seconds).
+    * Set the hello-multiplier to its default (typically 3) 
+    * Check that the ST values are reported correctly.
+* Adjusting Hello-Interval:
+    * Change the hello-interval to a different value.
+    * Verify that IS-IS adjacency is coming up.
+    * Verify that the updated Hello-Interval time is reflected in isis database output. 
```

Comment:
Hello-interval is not included in IS-IS LSPs.

Reply:
Updated from database to adjacency output